crates/uv/tests/sync.rs

```diff
@@ -55,6 +59,10 @@ fn locked() -> Result<()> {
         version = "0.1.0"
         requires-python = ">=3.12"
         dependencies = ["anyio==3.7.0"]
+
+        [build-system]
+        requires = ["setuptools>=42", "wheel"]
+        build-backend = "setuptools.build_meta"
```

Comment:
Unfortunately, we do need to set an explicit build backend for our test projects now. This is fine and correct, we were just omitting it before.